robot/vision.py

```diff
@@ -10,9 +10,9 @@
 from datetime import datetime
 from typing import NamedTuple, Any
 
-from robot.sheepdog_trials.markers import MARKER
-from robot.sheepdog_trials.teams import TEAM
-from .sheepdog_trials import BASE_MARKER as MarkerInfo
+from robot.marker_setup.markers import MARKER
+from robot.marker_setup.teams import TEAM
```

Comment:
Just to try and slim down the code, if this import isn't used then there is no need for it to be here.